google/cloud/bigtable/backup.py

```diff
@@ -339,6 +356,68 @@ def create(self, cluster_id=None):
             }
         )
 
+    def copy(
+        self,
+        new_backup_id,
+        project_id=None,
+        instance_id=None,
+        cluster_id=None,
+        expire_time=None,
+    ):
+        """Make a copy of this backup.
+
+        :type new_backup_id: str
+        :param new_backup_id: The name of the copied backup
+
+        :type project_id: str
+        :param project_id: The destination project id for the backup
+
+        :type instance_id: str
+        :param instance_id: The destination instance for the backup
+
+        :type cluster_id: str
+        :param cluster_id: The destination cluster for the backup
+
+        :type expire_time: :class:`datetime.datetime`
+        :param expire_time: (Optional) the new expiration time timestamp
+
+        :rtype: :class:`~google.api_core.operation.Operation`
+        :returns: :class:`~google.cloud.bigtable_admin_v2.types._OperationFuture`
+                  instance, to be used to poll the status of the 'copy' request
+        :raises Conflict: if the Backup already exists
+        :raises NotFound: if the Instance owning the Backup does not exist
+        :raises BadRequest: if the `table` or `expire_time` values are invalid,
+                            or `expire_time` is not set
+        """
+        if not expire_time:
+            expire_time = self._expire_time
+
+        if not project_id:
+            project_id = self._instance._client.project
+
+        if not instance_id:
+            instance_id = self._instance.instance_id
+
+        if not cluster_id:
+            cluster_id = self._cluster
+
+        api = self._instance._client.table_admin_client
+
+        parent = BigtableTableAdminClient.cluster_path(
+            project=project_id,
+            instance=instance_id,
+            cluster=cluster_id,
+        )
+
+        return api.copy_backup(
+            request={
+                "parent": parent,
```

Comment:
how does this work if the source backup project and parent backup project are different?

Reply:
`parent` is constructed using `project_id`, `instance_id`, `cluster_id`. It's allowed to be different I believe as long as the user have access to both. I'm not seeing any restriction about it from the design doc. From the client we will pass the path, and relay back error from the server.